catalog/security/catalog-security-pointofcontact-policyplugin/src/test/java/org/codice/ddf/catalog/security/policy/metacard/PointOfContactPolicyPluginTest.java

```diff
@@ -0,0 +1,212 @@
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.security.policy.metacard;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static ddf.catalog.Constants.OPERATION_TRANSACTION_KEY;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.data.impl.ResultImpl;
+import ddf.catalog.operation.OperationTransaction;
+import ddf.catalog.operation.Query;
+import ddf.catalog.operation.ResourceResponse;
+import ddf.catalog.operation.impl.ResourceRequestById;
+import ddf.catalog.plugin.PolicyResponse;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PointOfContactPolicyPluginTest {
+
+    public static final String TEST_POINT_OF_CONTACT = "test-point-of-contact";
+
+    public static final String TEST_ID = "test-id";
+
+    private PointOfContactPolicyPlugin pointOfContactPolicyPlugin =
+            new PointOfContactPolicyPlugin();
+
+    @Mock
+    private OperationTransaction mockOperationTransaction;
+
+    @Test
+    public void processPreCreateDoesNothing() throws java.lang.Exception {
+        PolicyResponse response = pointOfContactPolicyPlugin.processPreCreate(new MetacardImpl(),
+                Collections.emptyMap());
+
+        responseIsEmpty(response);
+    }
+
+    @Test
+    public void processPreUpdateDoesNothingWithNoPreviousMetacard() throws java.lang.Exception {
+        when(mockOperationTransaction.getPreviousStateMetacards()).thenReturn(Collections.emptyList());
+
+        Map<String, Serializable> inputProperties = setupAndGetInputProperties();
+        Metacard metacard = new MetacardImpl();
+
+        PolicyResponse response = pointOfContactPolicyPlugin.processPreUpdate(metacard,
+                inputProperties);
+
+        responseIsEmpty(response);
+    }
+
+    @Test
+    public void processPreUpdateDoesNothingWhenPointOfContactsAreSame() throws java.lang.Exception {
+        List<Metacard> listWithMetacard = new ArrayList<Metacard>();
+        MetacardImpl metacard = new MetacardImpl();
```

Comment:
✏️  Might be worth extracting the metacard creation logic to a `private` method, especially if we add multiple of them to the list (see next comment).